File: PHP/Tutorial.php
<?php

    require 'Binary.php';

    $name = '11010001';

    $res = LowLevel::reversed($name);
    echo('res now: ' . $res . '<br>');

    $lowLevel = LowLevel::NumberSystem_init();
    $res2 = $lowLevel->decimalToBinary('252');
    echo('res2: ' . $res2 . '<br>');

    $binary = new Binary();
    $res3 = $binary->modulus('11010','11');
    echo('res06: ' . $res3 . '<br>');

?>

File: Ruby/Tutorial.rb
require_relative "/home/pi/Downloads/ruby/LowLevelException.rb"
require_relative "/home/pi/Downloads/ruby/LowLevel.rb"
require_relative "/home/pi/Downloads/ruby/Number.rb"
require_relative "/home/pi/Downloads/ruby/Binary.rb"


puts "Hello, world!"

num = "11110010"
res = LowLevel.reversed(num)
puts("reversed is: " + res)

sys = LowLevel::NumberSystem.init()
number = sys.decimalToBinary("349")
puts("is: " + number)

list = Number.listNumberTypes()
puts("List is: " + list.to_s)

el = Binary.new()
num = el.smallest("0010011","11000")
puts("num1 is: " + num.to_s)